### DISCUSSION TITLE
my plugin for kong not work

### DISCUSSION THREAD

**Discussion by Canon4G** [Kong/kong#9267] [Help]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Kong version (`$ kong version`)

Kong 2.8.1.2-alpine

### Current Behavior

my kong work in docker

my plugin dir 
 /opt/share/kong/plugins/demo # ls
    handler.lua  schema.lua

my kong.conf in /etc/kong/kong.conf
  lua_package_path = /opt/share/?.lua;
  plugins = bundled, demo

but use this command in terminal curl http://localhost:8001 -s | jq .plugins | grep demo

no data response me

**Comment by ms2008** [comment]
I can also reproduce it locally with the official image, but there is no problem starting it with the local code, so there must be a bug somewhere.
